packages/flutter/test/widgets/implicit_animations_test.dart

```diff
@@ -315,7 +341,7 @@ void main() {
     await tester.tap(widgetFinder);
     await tester.pump();
     expect(mockOnEndFunction.called, 0);
-    await tester.pump(animationDuration);
+    await tester.pump(const Duration(milliseconds: 999));
     expect(mockOnEndFunction.called, 0);
     await tester.pump(additionalDelay);
     expect(mockOnEndFunction.called, 1);
```

Comment:
It used to be that with a `duration` of 1 second, the `onEnd()` callback would fire after `1.001` seconds. Not anymore!